src/cascadia/TerminalSettingsModel/FontConfig.cpp

```diff
@@ -30,6 +30,17 @@ winrt::com_ptr<FontConfig> FontConfig::CopyFontInfo(const FontConfig* source, wi
     MTSM_FONT_SETTINGS(FONT_SETTINGS_COPY)
 #undef FONT_SETTINGS_COPY
 
+    // We cannot simply copy the font axes and features with `fontInfo->_FontAxes = source->_FontAxes;`
+    // since that'll just create a reference; we have to manually copy the values.
+    if (source->_FontAxes)
+    {
+        fontInfo->_FontAxes = winrt::single_threaded_map<winrt::hstring, float>();
+        for (const auto keyValuePair : source->_FontAxes.value())
+        {
+            fontInfo->_FontAxes.value().Insert(keyValuePair.Key(), keyValuePair.Value());
+        }
+    }

```

Comment:
This makes the `MTSM_FONT_SETTINGS` macro call above pointless for `FontAxes`, but I'd still rather not remove `FontAxes` from that macro since it is relevant in other places